### DISCUSSION TITLE
Display correct answers (and answer formats) for the quizzes

### DISCUSSION THREAD

**Discussion by eraunio** [pykale/transparentML#55] [Suggestions]
It would be a good idea to have a separate page to review the correct answers for the quizzes. Currently, all we can know is whether an answer is correct or not, which is not necessarily so helpful.

Also, as Google Forms seems to be very sensitive to the answer formats, it would be great to accompany each question with an example answer and possible rounding. For example, a percentage with two decimal places: 15.687976978 > 15.68%

**Comment by haipinglu** [comment]
> It would be a good idea to have a separate page to review the correct answers for the quizzes. Currently, all we can know is whether an answer is correct or not, which is not necessarily so helpful.
> 
Thanks for your feedback! We provide and review correct answers for all exercises but not the quizzes. Providing answers will turn the quizzes into exercises because based on our experiences, for many learners, resisting the temptation to check the answers is hard. Exercises serve the purpose that you prefer where we hide the answers. I suggest you to work on the exercises as if they are quizzes and then check the answers provided.

> Also, as Google Forms seems to be very sensitive to the answer formats, it would be great to accompany each question with an example answer and possible rounding. For example, a percentage with two decimal places: 15.687976978 > 15.68%

Yes, this is our usual practice and we think we did it for all quiz questions, e.g. Quiz 2 Q9 says "9. Find the minimum RSS using the weight and bias of the trained model from Question 8 (*to the nearest 2 decimal places, e.g. 1.23*). " If you see any questions where we should but we did not do so, please point it out and we will act promptly to fix it. 

Highly appreciate your feedback. Thank you!
